types/zengin-code/zengin-code-tests.ts

```diff
@@ -0,0 +1,23 @@
+import zenginCode from 'zengin-code';
```

Comment:
```suggestion
import zenginCode = require('zengin-code');
```
https://github.com/zengin-code/zengin-js#usage
ESM syntax is still supported (via default ts configuration)